### DISCUSSION TITLE
Error (Xcode): Multiple commands produce "xxxxxxxxxxx"

### DISCUSSION THREAD

**Discussion by MICHEL-ADJE-GITHUB** [codemagic-ci-cd/codemagic-docs#2181] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I'm trying to deploy my app to the Apple Store using Codemagic since I don't have a Macbook
I'm follow all steps and finalement j'ai cette erreur lors de la construition de mon app ios

### Expected results

all works good

### Actual results

Error (Xcode): Multiple commands produce '/Users/builder/Library/Developer/Xcode/DerivedData/Runner-edaimyiflreloheqntgnhkmwcclv/Build/Intermediates.noindex/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/Reachability.framework'

**Comment by dtrdic** [comment]
hey @MICHEL-ADJE-GITHUB please share a build id so i can take a closer look

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion due to a lack of activity. You can open the discussion again by replying to this message or opening a new discussion.

**Reply by the-infinite** [reply]
Hi @dtrdic. I am experiencing the same issue right now. My build ID is 65cf5228853363e7bb67a5a0.

Please, could you help me check it out?

**Reply by dtrdic** [reply]
hey @the-infinite can you check [this suggestion](https://github.com/AngeloAvv/flutter_flavorizr/issues/46#issuecomment-968896934) form github dicussion if it helps, seems to be a common issue

**Reply by the-infinite** [reply]
Oh-kay, thank you. I'd check it out and get back to you.

**Reply by vicktor-kiprotich** [reply]
> Oh-kay, thank you. I'd check it out and get back to you.

Hi @the-infinite , how did you resolve this issue?
